src/ggml-backend-impl.h

```diff
@@ -44,7 +44,7 @@ extern "C" {
         // base address of the buffer
         void *       (*get_base)     (ggml_backend_buffer_t buffer);
         // (optional) initialize a tensor in the buffer (eg. add tensor extras)
-        void         (*init_tensor)  (ggml_backend_buffer_t buffer, struct ggml_tensor * tensor);
+        enum ggml_status  (*init_tensor)  (ggml_backend_buffer_t buffer, struct ggml_tensor * tensor);
```

Comment:
All backends that use this function will need to be updated. It would be preferable to open the PR in llama.cpp since it has much better CI.

Reply:
ok. To move forward step after step, would you accept a PR in llamacpp  with just that init_tensor change?

Reply:
Yes.